azure-kusto-data/azure/kusto/data/exceptions.py

```diff
@@ -49,8 +49,8 @@ def __init__(self, authentication_method: str, exception: Exception, **kwargs):
         self.exception = exception
         if "authority" in kwargs:
             self.authority = kwargs["authority"]
-        if "resource" in kwargs:
-            self.kusto_cluster = kwargs["resource"]
+        if "kusto_uri" in kwargs:
+            self.kusto_cluster = kwargs["kusto_uri"]
```

Comment:
This is technically a breaking change, no? Someone who inspects the exception will suddenly get none.
Maybe we can keep both for now and remove "resource" in the next major.

Reply:
I got a comment from a customer this is not an intuitive name if the value is missing or corrupt (which was the case with his problem).
@ArielYehezkely , how do you feel about this change?

Reply:
I don't think we need to worry about it, its not an API and I don't think people actually base their code on such members, people usually just print/log the exception.
If you want to be extra cautious you can add instead of replace but please open a PR which removes the old one with "merge with next major version bump" label.